__tests__/app.test.js

```diff
@@ -88,9 +87,50 @@ describe("app", () => {
         expect(body.msg).toBe("Bad request");
       });
   });
-  test("404: GET responds with correct message for valid but non-existant article_ids", () => {
+  test("404: GET responds with correct message for valid but non-existent article_ids", () => {
+    return request(app)
+      .get("/api/articles/1000")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("not found");
+      });
+  });
+  test("200: GET - Responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toEqual([
+          {
+            article_id: 3,
+            author: "icellusedkars",
+            body: "Ambidextrous marsupial",
+            comment_id: 11,
+            created_at: "2020-09-19T23:10:00.000Z",
+            votes: 0,
+          },
+          {
+            article_id: 3,
+            author: "icellusedkars",
+            body: "git push origin master",
+            comment_id: 10,
+            created_at: "2020-06-20T07:24:00.000Z",
+            votes: 0,
+          },
```

Comment:
Needs a 200 test to check that it returns empty array if article has no comments